easybuild/easyconfigs/t/torchaudio/torchaudio-2.2.0-foss-2023a-PyTorch-2.1.2-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PythonPackage'
```

Comment:
Looks like the same as #19978 , but just for reference, see comment on: https://github.com/easybuilders/easybuild-easyconfigs/pull/19987 for using PyTorch-bundle instead of torchaudio